keyboards/ok60/keymaps/default/keymap.c

```diff
@@ -7,7 +7,7 @@ const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
         KC_TAB,  KC_Q,    KC_W,    KC_E,  KC_R,  KC_T,  KC_Y,  KC_U,  KC_I,    KC_O,    KC_P,    KC_LBRC, KC_RBRC, KC_BSLS,
         KC_CAPS, KC_A,    KC_S,    KC_D,  KC_F,  KC_G,  KC_H,  KC_J,  KC_K,    KC_L,    KC_SCLN, KC_QUOT, KC_ENT,
         KC_LSFT, KC_Z,    KC_X,    KC_C,  KC_V,  KC_B,  KC_N,  KC_M,  KC_COMM, KC_DOT,  KC_SLSH, KC_LSFT,
-        KC_LCTL, KC_LGUI, KC_LALT,               KC_SPC,                       KC_LALT, MO(1),   KC_MENU, KC_LCTL
+        KC_LCTL, KC_LGUI, KC_LALT,               KC_SPC,                       KC_RALT, MO(1),   KC_MENU, KC_RCTL
```

Comment:
Could you change `KC_MENU` to `KC_APP` here? That's actually the key most people would expect to be there.

Everything besides that looks great.

```suggestion
        KC_LCTL, KC_LGUI, KC_LALT,               KC_SPC,                       KC_RALT, MO(1),   KC_APP,  KC_RCTL
```